projects/packages/waf/composer.json

```
@@ -4,7 +4,8 @@
 	"type": "jetpack-library",
 	"license": "GPL-2.0-or-later",
 	"require": {
-		"wikimedia/aho-corasick": "^1.0"
+		"wikimedia/aho-corasick": "^1.0",
+		"automattic/jetpack-status": "@dev"
```

Optional, but it tends to be nicer if they're in alphabetical order.
```suggestion
		"automattic/jetpack-status": "@dev",
		"wikimedia/aho-corasick": "^1.0"
```